posawesome/posawesome/api/offline_sync/common.py

```
@@ -0,0 +1,67 @@
+
+def _resolve_profile(pos_profile=None):
+	if isinstance(pos_profile, dict):
+		return pos_profile
+
+	if isinstance(pos_profile, str):
+		raw_value = pos_profile.strip()
+		if not raw_value:
+			return get_active_pos_profile()
+		try:
+			decoded = json.loads(raw_value)
+		except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)